assets/xml/objects/object_st.xml

```diff
@@ -34,8 +34,8 @@
         <DList Name="object_st_DL_0048B8" Offset="0x48B8"/>
         <DList Name="object_st_DL_004950" Offset="0x4950"/>
         <DList Name="object_st_DL_0049E8" Offset="0x49E8"/>
-        <DList Name="object_st_DL_004DB0" Offset="0x4DB0"/>
-        <DList Name="object_st_DL_004EB8" Offset="0x4EB8"/>
+        <DList Name="gSkulltulaTokenDL" Offset="0x4DB0"/>
+        <DList Name="gSkulltulaTokenFlameDL" Offset="0x4EB8"/>
```

Comment:
Looking at those DLists, I think both *are* the token, the first one has the base token (flat cylinder shaped), the second has the flame effect as well as the eyes/mouth thing I'm pretty sure

Reply:
`gSkulltulaTokenDL` / `gSkulltulaTokenEmbellishmentsDL` :4Head: